src/client/theme-default/components/VPBadge.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+const props = withDefaults(
+    defineProps<{
```

Comment:
Should use 2-spaces indent..?

(prettier didn't take effect? 🤔 )

Reply:
Yeah, prettier is actually disabled for vue. It messes up the ending tags. This needs to be manually formatted, or just comment *.vue in prettierignore and format this one, remove that comment then fix the tags.

Reply:
OK, now I haved formmatted it to 2-spaces manually